src/App.tsx

```diff
@@ -5,7 +5,21 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [isMovieNew, setIsMovieNew] = useState(true);
+
+  const addMovie = (movie: Movie) => {
+    const isExistMovie = movies.find(
+      existMovie => existMovie.imdbId === movie.imdbId,
+    );
+
+    if (!isExistMovie) {
+      setIsMovieNew(true);
+      setMovies((prevMovies) => [...prevMovies, movie]);
+    } else {
+      setIsMovieNew(false);
+    }
```

Comment:
simplify condition
```suggestion
      setIsMovieNew(!isExistMovie);
      setMovies((prevMovies) => [...prevMovies, movie]);
```

Reply:
Would like to shorter the way the show but unfortunately it can break the code, because short in this way leads lack of some settings as well

Reply:
```
  setIsMovieNew(!isExistMovie);
  if (!isExistMovie) {
      setMovies((prevMovies) => [...prevMovies, movie]);
  }
```